src/app/core/nav-menu/routes/app-nav-routes.ts

```diff
@@ -0,0 +1,20 @@
+import { Route } from '../models';
+import { virtualMachinesSubroutes } from './virtual-machines-subroutes';
+import { accountsSubroutes } from './accounts-subroutes';
+
+export const appNavRoutes: Route[] = [
+  {
+    id: 'virtual-machines',
+    text: 'NAVIGATION_SIDEBAR.VMS',
+    path: '/instances',
+    icon: 'mdi-cloud',
+    subroutes: virtualMachinesSubroutes,
```

Comment:
Direct links to the other array can cause errors in case of mutation. Suggest change to `subroutes: [...virtualMachinesSubroutes]`

Reply:
Is it will be helpful if we don't get a deep copy of the array?